eng/pipelines/templates/CrossFramework_Tests_On_Windows.yml

```diff
@@ -1,4 +1,11 @@
 steps:
+- task: UseDotNet@2
+  displayName: 'Use .NET Core sdk'
+  inputs:
+    packageType: sdk
+    version: 6.x
+    installationPath: $(Agent.TempDirectory)/dotnet
```

Comment:
Any reason not to let the task use its own default installation path?

Reply:
According to [doc](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops#task-inputs):
```
$(Agent.ToolsDirectory): This makes the version to be cached on the agent since this directory is not cleanup up across pipelines. All pipelines running on the agent, would have access to the versions installed previously using the agent.
$(Agent.TempDirectory): This can ensure that a pipeline doesn't use any cached version of .NET core since this folder is cleaned up after each pipeline.
Default value: $(Agent.ToolsDirectory)/dotnet
```
The default value $(Agent.ToolsDirectory)/dotnet will not be cleanup across pipelines. But $(Agent.TempDirectory) will be cleaned up after each pipeline. I think we do not want to pollute the agent machine to affect others' builds.